src/commands/infra/generate.ts

```diff
@@ -353,13 +360,15 @@ export const generateConfigWithParentEqProjectPath = async (
     writeTfvarsFile(spkTfvarsObject, parentDirectory, SPK_TFVARS);
     await copyTfTemplate(templatePath, parentDirectory, true);
   } else {
+    // TOFIX: are these warnings? if yes, please write some comments.
     logger.warn(`Variables are not defined in the definition.yaml`);
   }
   if (parentInfraConfig.backend) {
     const backendTfvarsObject = generateTfvars(parentInfraConfig.backend);
     checkTfvars(parentDirectory, BACKEND_TFVARS);
     writeTfvarsFile(backendTfvarsObject, parentDirectory, BACKEND_TFVARS);
   } else {
+    // TOFIX: are these warnings? if yes, please write some comments.
```

Comment:
This is also another valid warning.

Reply:
Not all templates will require a remote backend (i.e Bedrock's azure-simple). Keep in mind, that if a remote backend is not configured for a template, it will nearly impossible to be able to use `spk infra` in a pipeline, but this can still work locally.